### PR TITLE
Add README file about legoAdapter

### PR DESCRIPTION
**Pull request by eikhr**
# Description

Adds some documentation explaining what `legoAdapter` does, and how to use it. Also fixes a few type-errors.

### CODE REVIEW

**Review by ivarnakken — APPROVED** [review]
I like!😍

**Review by Bestem0r — APPROVED** [review]
🙏

File: app/utils/legoAdapter/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- `selectIds`: Get the IDs array
+
+## How do I use it?
+
+Take a look at an example slice: [emailLists](../../reducers/emailLists.ts)
+
+When initializing the adapter with entityType:
+
+```typescript
+const emailListSlice = createLegoAdapter(EntityType.EmailLists);
```
Comment (norbye) [inline app/utils/legoAdapter/README.md:44]:
```suggestion
const legoAdapter = createLegoAdapter(EntityType.EmailLists);
```

**Review by norbye — APPROVED** [review]
noice

**Review by jonasdeluna — APPROVED** [review]
Awesome

**Pull request merged**